src/database/database-sqlite3.cpp

```diff
@@ -779,3 +779,105 @@ void AuthDatabaseSQLite3::writePrivileges(const AuthEntry &authEntry)
 		sqlite3_reset(m_stmt_write_privs);
 	}
 }
+
+ModMetadataDatabaseSQLite3::ModMetadataDatabaseSQLite3(const std::string &savedir):
+	Database_SQLite3(savedir, "mod_storage"), ModMetadataDatabase()
+{
+}
+
+ModMetadataDatabaseSQLite3::~ModMetadataDatabaseSQLite3()
+{
+	FINALIZE_STATEMENT(m_stmt_remove)
+	FINALIZE_STATEMENT(m_stmt_set)
+	FINALIZE_STATEMENT(m_stmt_get)
+}
+
+void ModMetadataDatabaseSQLite3::createDatabase()
+{
+	assert(m_database); // Pre-condition
+
+	SQLOK(sqlite3_exec(m_database,
+		"CREATE TABLE IF NOT EXISTS `pairs` (\n"
```

Comment:
why "pairs"? Is this term used elsewhere? "entry" would make more sense, it's more commonly used by key-value dictionaries

Reply:
Good point. I will change the name of the table to "entries".

Reply:
I'm leaning towards `entry` too, but considering that the Lua table iterator is called `pairs` and not `entries`, I consider `pairs` fine as more Lua-esque terminology.

Reply:
If I change the name of the table, should I call it "entry" or "entries"? The plural form seems to be consistent with most of the other SQLite table names used.